lib/v2/ruijuzi/feeds.js

```diff
@@ -0,0 +1,51 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const iconv = require('iconv-lite');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const base_url = `http://rui.juzi.bot`;
```

Comment:
- Please use `lowerCamelCase`
- Use HTTPS
- Unnecessary use of template literals. Only use it if the string contains an expression. Please use single quotes instead.